packages/config/config/devices/0x010f/fgwceu-201.json

```diff
@@ -0,0 +1,183 @@
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"label": "FGWCEU-201",
+	"description": "WALLI FIBARO CONTROLLER",
+	"devices": [
+		{
+			"productType": "0x2301",
+			"productId": "0x1000"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
```

Comment:
We don't use that property anymore. Is that auto-generated or are you adding it?
```suggestion
```

Reply:
I was adding it. Will not add it anymore